obspy/io/seiscomp/inventory.py

```diff
@@ -267,10 +269,20 @@ def _read_station(instrumentation_register, sta_element, _ns):
                                datum=True)
     elevation = _read_floattype(sta_element, _ns("elevation"), Distance,
                                 unit=True)
+    public_id = sta_element.get("publicID")
+    ns = "seiscompml"
+    extra_tag = AttribDict({})
+    extra_tag.public_id = AttribDict({
+            'value': public_id,
+            'namespace': ns
+        }
+    )
```

Comment:
There are a few things that should be changed here I think.

#### use namespaces correctly

Namespace should be set to the actual namespace value of the attribute `publicID` that is being parsed and not quite arbitrarily `"seiscmopml"`. There is an easy way that might be doing the right thing..

```
namespace = sta_element.nsmap[None]
```

... but this is working on assumptions how namespaces are set in sc3ml and might break later on.
So to correctly determine the namespace of this attribute, you should check the attribute for a namespace first and if it doesn't have one it should be OK to use the default namespace of it's element

```python
match = re.match(r'{([^}]*)}.*', sta_element.tag)
if match:
    namespace = match.group(1)
else:
    namespace = sta_element.nsmap[None]
```

I haven't messed around with details of lxml and xml namespaces in a while, but it seems that the default namespace is set on the station element, even though the whole document is operating on the implicit default namespace declared on the root element, so it should be OK to assume we can use that.

Actually, as a last thought.. when accessing `public_id = sta_element.get("publicID")` that is already assuming that `publicID` has the default namespace of the station item, I believe, so maybe just using station default namespace is fine..

#### naming of extra item

I would probably keep the name `publicID` as the name of the item in extra. Even though that means internally you would look at `station.extra.publicID` which isn't in line with our Python object naming scheme.. this a) makes it clear that this is not coming from Python land and b) if writing an inventory read from sc3ml as stationxml, your custom attribute would kind of properly show up in the xml file as something like..

```xml
<FDSNStationXML xmlns="http://www.fdsn.org/xml/station/1" schemaVersion="1.2">
  <Source>scxml import</Source>
  <Sender>ObsPy Inventory</Sender>
  <Module/>
  <ModuleURI/>
  <Created>2022-12-01T12:14:43.872676Z</Created>
  <Network code="EB" startDate="1980-01-01T00:00:00.000000Z" restrictedStatus="open">
    <Description>SINGLE STATION</Description>
    <Station xmlns:ns0="seiscompml" code="EBR" startDate=..... ns0:public_id="Station/EB/EBR/2002-04-01T00:00:00.0000Z">
      <Latitude unit="DEGREES">40.820599</Latitude>
      <Longitude unit="DEGREES">0.4933</Longitude>
...
```

#### should set type as attribute

Then it would come out as an attribute on StationXML output as opposed to an element.